### PR TITLE
Email Notifications

### PR DESCRIPTION
**Pull request by amyshan**
## Email Notifications
Emails are sent when:
- Requests are opened
- Requests are closed
- Requests are completed
- Commissions are opened

### Related PRs
#43 - Calvin more or less did the request mailers he just had a tiny bug in one that I fixed

### Migrations
n/a

### Tests Performed, Edge Cases
- Opened, closed, and archived Requests
- Opened commissions

### Screenshots
The emails look more or less the same here's just a confirmation that they exist:
image

CC: @by-co

### CODE REVIEW

File: app/views/commission_mailer/new_commission_email.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <body>
+    <h1>Commission Opened</h1>
+    <p>
+      <%= @buyer.name %> has opened a new commission for you. <br>
+    </p>
+    <h2>
+      Contact Info:
+    </h2>
+    <p>
+      Email: <%= @buyer.name %> <br>
```
Comment (by-co) [inline app/views/commission_mailer/new_commission_email.html.erb]: Typo?
```suggestion
      Email: <%= @buyer.email %> <br>
```

Comment (by-co) [reply app/views/commission_mailer/new_commission_email.html.erb]: Same typo in the requests email too
